pkgs/applications/networking/misc/zammad/default.nix

```
@@ -106,7 +85,7 @@ stdenv.mkDerivation {
     rubyEnv
     rubyEnv.wrappedRuby
     rubyEnv.bundler
-    yarn
+    pnpm.configHook
     nodejs
     procps
     cacert
```

```suggestion
  nativeBuildInputs = [
    pnpm.configHook
    nodejs
    procps
    cacert
```

I just thought about the `buildInputs` and suspected that these inputs could actually be `nativeBuildInputs`.

Based on this PR, I moved these inputs to the `nativeBuildInputs` and deployed my test Zammad with them and it works as expected. The NixOS test also runs without any problems when these inputs are moved to the `nativeBuildInputs`.